src/seeded_rng.rs

```diff
@@ -107,7 +105,6 @@ impl SeededRng {
                 *seed_offset += 1;
                 Rc::new(RefCell::new(SeededRng {
                     seed: seed,
-                    print_seed_on_drop: false,
```

Comment:
I think we actually just should set `seed` to the global `SEED` instead of the one that we pass into `XorShiftRng::from_seed`, then it should be fine.